index.html

```diff
@@ -177,6 +173,7 @@ <h2>Where?</h2>
           <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3054.691317767825!2d-76.30661888384205!3d40.03767078652806!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c624edea052ac9%3A0xc7ceadc87b895d3b!2sPubforge!5e0!3m2!1sen!2sus!4v1483977317358" width="750" height="350" frameborder="0" style="border:0" allowfullscreen></iframe>
           -->
 
+<<<<<<< HEAD
```

Comment:
This is an artifact of the merge conflict. Please resolve the conflict and remove these `<<<< >>>>` comments